src/pages/Memories.tsx

```diff
@@ -84,31 +86,40 @@ const MemoriesPage = () => {
                 </div>
               );
             })}
-          </>
+          </div>
         ) : (
           <>
-            {data?.map((col, inx) => {
-              return (
-                <div className="flex flex-col h-fit gap-4" key={inx}>
-                  {col.map((cardData: any) => {
-                    return (
-                      <MemoryCard
-                        key={cardData?.imgUrl}
-                        tags={cardData?.tags}
-                        title={cardData?.title}
-                        message={cardData?.message}
-                        imageUrl={cardData?.imgUrl}
-                        author={cardData?.name || "Anon"}
-                      />
-                    );
-                  })}
-                </div>
-              );
-            })}
+             <ParallaxScroll className=" w-full h-full" images={[
```

Comment:
@PYAG1 you are still using the images you added for showcase oo, could you change it so it uses the cards?